cmd/crates/soroban-spec-typescript/src/project_template/src/invoke.ts

```diff
@@ -102,38 +103,30 @@ export async function invoke<R extends ResponseTypes, T = string>({
     fee: fee.toString(10),
     networkPassphrase,
   })
+    .setNetworkPassphrase(networkPassphrase)
```

Comment:
Isn't this added two lines above when initializing the builder?
```suggestion
```

Reply:
uhh yeah, dunno why that happened